static/app/gettingStartedDocs/java/log4j2.tsx

```diff
@@ -125,7 +125,10 @@ SENTRY_PROPERTIES_FILE=sentry.properties java -javaagent:sentry-opentelemetry-ag
 `;
 
 const getSentryPropertiesSnippet = (params: Params) => `
-dsn=${params.dsn.public}${
+dsn=${params.dsn.public}
+# Add data like request headers and IP for users,
+# see https://docs.sentry.io/platforms/java/guides/log4j2/data-management/data-collected/ for more info
+send-defaut-pii=true${
   params.isPerformanceSelected
     ? `
 traces-sample-rate=1.0`
```

Comment:
There's still a sample without `sendDefaultPii` here. While it would be possible to change the SDK and add that option to the XML config I think it's better to just use `sentry.properties` instead as that already existed for a while and doesn't require use of a new version of the SDK.